pilot/pkg/networking/core/cluster_waypoint.go

```diff
@@ -70,11 +74,17 @@ func buildInternalUpstreamCluster(name string, internalListener string) *cluster
 
 var (
 	GetMainInternalCluster = func() *cluster.Cluster {
-		return buildInternalUpstreamCluster(MainInternalName, MainInternalName)
+		return buildInternalUpstreamCluster(MainInternalName, MainInternalName, true)
 	}
 
-	GetEncapCluster = func() *cluster.Cluster {
-		return buildInternalUpstreamCluster(EncapClusterName, ConnectOriginate)
+	GetEncapCluster = func(p *model.Proxy) *cluster.Cluster {
```

Comment:
hmm is this worth breaking out... since its not encapsulating, the name seems pretty misleading.. its just a raw forwarding?

Why do we need internal upstream anyways for eastwest if we are not tunneling?

Reply:
The reason I kept the forwarding logic is because the existing clusters all send traffic to encap; I didn't want to create duplicate clusters with slightly different names that sent traffic to the real upstreams